src/layout-persistence/dotnet/tests/MorganStanley.ComposeUI.LayoutPersistence.Tests/JsonLayoutSerializerTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿/*
+ * Morgan Stanley makes this available to you under the Apache License,
+ * Version 2.0 (the "License"). You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Unless required by applicable law or agreed
+ * to in writing, software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+using System.Text.Json;
+using FluentAssertions;
+using MorganStanley.ComposeUI.LayoutPersistence.Serializers;
+
+namespace MorganStanley.ComposeUI.LayoutPersistence.Tests;
+
+public class JsonLayoutSerializerTests
+{
+    private readonly JsonLayoutSerializer _serializer = new JsonLayoutSerializer();
+
+    [Fact]
+    public async Task Serialize_ShouldReturnValidJsonString()
```

Comment:
I know that sometimes I have unpopular opinions regarding tests, so take this with a grain of salt, but:
IMHO, this is only really testing the System.Text.Json.JsonSerializer - something that we shouldn't do.

I would only test that using the serializer and reading the data back results in an equivalent object